packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -0,0 +1,279 @@
+                                  <th className={css['table__cell--center']}>Uptime</th>
+                                </tr>
+                              </thead>
+                              <tbody>
+                                {group.validators.map((validator, j) => (
+                                  <tr key={`${i}.${j}`}>
+                                    <td className={css['table__cell--title']}>{validator.name}</td>
+                                    <td className={css['table__cell--center']}>
+                                      <span
+                                        className={[
+                                          css.circle,
+                                          css[`circle--${validator.elected ? 'ok' : 'error'}`],
+                                        ].join(' ')}
+                                      />
+                                    </td>
+                                    <td className={css['table__cell--center']}>
+                                      <span
+                                        className={[
+                                          css.circle,
+                                          css[`circle--${validator.online ? 'ok' : 'error'}`],
+                                        ].join(' ')}
+                                      />
+                                    </td>
+                                    <td>
+                                      {this.cutAddress(validator.address)}
+                                      <span
+                                        className={css.copy}
+                                        onClick={this.copyAddress.bind(this, validator.address)}
+                                      >
+                                        copy
+                                      </span>
+                                    </td>
+                                    <td className={css['table__cell--center']}>
+                                      {this.formatNumber(validator.usd, 2)}
+                                    </td>
+                                    <td className={css['table__cell--center']}>
+                                      {this.formatNumber(validator.gold, 2)}
+                                    </td>
+                                    <td className={css['table__cell--center']}>
+                                      {this.formatNumber(validator.uptime, 1)}%
+                                    </td>
+                                  </tr>
+                                ))}
+                              </tbody>
+                            </table>
+                          )}
+                        </div>
+                      </td>
+                    </tr>
+                  )}
+                </>
+              ))}
+            </tbody>
+          </table>
+        </View>
+      </View>
+    )
+  }
+}
+
+export default withNamespaces('dev')(ValidatorsListApp)
```

Comment:
preferable to use the `Namespaces.dev` constant

Reply:
Done, thanks!